File: bench/cases/array-sum-1m/main.go
package main

import "fmt"

func main() {
	xs := make([]int64, 0)
	for i := int64(0); i < 10000000; i++ {
		xs = append(xs, i)
	}
	var sum int64 = 0
	for j := 0; j < len(xs); j++ {
		sum += xs[j]
	}
	fmt.Println(sum)
}

File: bench/cases/array-sum-1m/main.ts
let xs: number[] = []
let i = 0
while (i < 10000000) {
  xs.push(i)
  i = i + 1
}
let sum = 0
let j = 0
while (j < xs.length) {
  sum = sum + xs[j]
  j = j + 1
}
console.log(sum)
